vetiver/handlers/base.py

```diff
@@ -79,19 +79,20 @@ def __init__(self, model, ptype_data):
 
     def describe(self):
         """Create description for model"""
-        desc = f"{self.model.__class__} model"
+
+        pip_name = self.pip_name if hasattr(self, "pip_name") else ""
+        obj_name = type(self.model).__qualname__
+
+        desc = f"A {pip_name} {obj_name} model"
+
         return desc
 
-    def create_meta(
-        user: list = None,
-        version: str = None,
-        url: str = None,
-        required_pkgs: list = [],
-    ):
+    def create_meta(self, metadata):
         """Create metadata for a model"""
-        meta = _model_meta(user, version, url, required_pkgs)
 
-        return meta
+        pip_name = self.pip_name if hasattr(self, "pip_name") else None
```

Comment:
to handle the `pkg` weirdness, the `pkg` attribute was removed. if there is a pip name, it will be handled as expected, otherwise it will set to None.